actions/workflows/st2_pkg_upgrade_e2e_test.yaml

```diff
@@ -230,32 +188,32 @@ tasks:
               versions: <% result().output.versions %>
               version_str: <% result().output.versions.items().select( $[0] + "=" + $[1]).join("\n\t") %>
         do:
-          - create_vm_windows
-  create_vm_windows:
-    action: st2cd.create_vm_windows
-    input:
-      hostname: <% ctx().windows_hostname %>
-      instance_type: <% ctx().instance_type %>
-      environment: <% ctx().environment %>
-      dns_zone: <% ctx().dns_zone %>
-      distro: <% ctx().windows_distro %>
-      role: <% ctx().role %>
-    next:
-      - when: <% succeeded() %>
-        publish:
-          - vm_windows_info: <% result().output.vm_info %>
-          - vm_windows_id: <% result().output.vm_info.id %>
-          - vm_windows_username: <% result().output.vm_username %>
-          - vm_windows_password: <% result().output.vm_password %>
-        do:
+          # - create_vm_windows
```

Comment:
Similar to above, perhaps a comment to state to remove the vars for these parameters when re-instate windows tests